packages/amplication-data-service-generator/src/server/static/src/auth/auth.module.ts

```diff
@@ -7,7 +7,7 @@ import { UserModule } from "../user/user.module";
 import { AuthController } from "./auth.controller";
 import { AuthResolver } from "./auth.resolver";
 import { AuthService } from "./auth.service";
-import { BasicStrategy } from "./basic.strategy";
+import { BasicStrategy } from "./basic/basic.strategy";
```

Comment:
why do we need to import basicStrategy, and we do not need JwtStartegy?

If it is required, we probably need to add JwtStartegy. if not - please open a new issue and remove this line of code in a separate PR

Reply:
Look at the line below, we do import the JwtStrategy